src/test/uint256_tests.cpp

```diff
@@ -166,12 +166,14 @@ BOOST_AUTO_TEST_CASE( methods ) // GetHex SetHex begin() end() size() GetLow64 G
     uint256 TmpL(R1L);
     BOOST_CHECK_EQUAL(TmpL, R1L);
     // Verify previous values don't persist when setting to truncated string.
-    TmpL.SetHex("21");
+    TmpL.SetHexDeprecated("21");
     BOOST_CHECK_EQUAL(TmpL.ToString(), "0000000000000000000000000000000000000000000000000000000000000021");
-    TmpL.SetHex(R2L.ToString());   BOOST_CHECK_EQUAL(TmpL, R2L);
-    TmpL.SetHex(ZeroL.ToString()); BOOST_CHECK_EQUAL(TmpL, uint256());
+    TmpL = *uint256::FromHex(R2L.ToString());
+    BOOST_CHECK(TmpL == R2L);
+    TmpL = *uint256::FromHex(ZeroL.ToString());
+    BOOST_CHECK(TmpL == uint256());
```

Comment:
Should use `BOOST_CHECK_EQUAL_COLLECTIONS()`.

Reply:
Yeah, sorry, `BOOST_CHECK_EQUAL()`. Seems to be a mis-merge on your part when rebasing. Considering I took the time to add a whole extra commit to convert away from `BOOST_CHECK` in this file recently, I would be quicker to ACK if you fixed the merge.

Reply:
+1, gives way better errors on failure